packages/web-app-files/src/components/SideBar/Links/FileLinks.vue

```diff
@@ -89,11 +89,16 @@ export default {
   title: $gettext => {
     return $gettext('Links')
   },
+  provide() {
+    return {
+      changeView: view => (this.$data.CURRENT_VIEW = view)
+    }
+  },
   data() {
     return {
-      // panel types
-      PANEL_SHOW: PANEL_SHOW,
-      PANEL_EDIT: PANEL_EDIT
+      VIEW_SHOW,
+      VIEW_EDIT,
+      CURRENT_VIEW: VIEW_SHOW
```

Comment:
Same as with the other one, `currentView` is not a constant. Would like to keep the naming as camelCase.